src/DotNetLightning.Core/Utils/SeqParser.fs

```diff
@@ -0,0 +1,109 @@
+namespace DotNetLightning.Utils
+
+open ResultUtils
+open ResultUtils.Portability
+
+// A SeqParser is a parser for sequences. It wraps a function which takes a
```

Comment:
nit: comments like this is better written with `///` rather than `//`. See https://docs.microsoft.com/en-us/dotnet/fsharp/language-reference/xml-documentation#comments-without-xml-tags